mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/domain/dto/ArenaMigreringTiltaksgjennomforingDto.kt

```diff
@@ -32,7 +32,10 @@ data class ArenaMigreringTiltaksgjennomforingDto(
     val deltidsprosent: Double,
 ) {
     companion object {
-        fun from(tiltaksgjennomforing: TiltaksgjennomforingAdminDto, arenaId: Int?, endretTidspunkt: LocalDateTime) =
+        fun from(tiltaksgjennomforing: TiltaksgjennomforingAdminDto, arenaId: Int?, endretTidspunkt: LocalDateTime) {
+            // Alle fra arena har arena_ansvarlig_enhet og nav_region er required fra admin-flate
+            requireNotNull(tiltaksgjennomforing.navRegion ?: tiltaksgjennomforing.arenaAnsvarligEnhet)
```

Comment:
Kan slenge inn en message for `requireNotNull` som gir en forståelig feilmelding dersom metoden kaster en feil. Kan legges på slik
```kt
requireNotNull(tiltaksgjennomforing.navRegion ?: tiltaksgjennomforing.arenaAnsvarligEnhet) {
    "Message her"
}
```